cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -427,7 +427,7 @@ def _dna_cases_related_to_dna_sample(
     ) -> list[str]:
         """Maps a list of uploaded DNA cases linked to DNA sample."""
         potential_cases_related_to_dna_sample: list[Case] = [
-            dna_sample_family_relation.family for dna_sample_family_relation in dna_sample.links
+            dna_sample_family_relation.case for dna_sample_family_relation in dna_sample.links
```

Comment:
```suggestion
            dna_sample_case_relation.case for dna_sample_case_relation in dna_sample.links
```

Reply:
I think this commit suggestion would be contained and safe.

Reply:
Me too!